message.go

```diff
@@ -36,6 +36,23 @@ const (
 )
 
 type Props map[string]string
+
+func (props *Props) GetIntValue(key string) (int64, error) {
```

Comment:
You attach the method to a type with pointer reference and then you dereference to access the map key.
Just don't. Change method signature to 

func (p Props) GetIntValue(key string) (int64, error)

so you will be able to access value without dereferencing

p[key]